modules/local/propr/propd/main.nf

```diff
@@ -0,0 +1,33 @@
+process PROPR_PROPD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    // conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/bioconductor-limma_r-ggplot2_r-propr:209490acb0e524e3' :
```

Comment:
We need to use the https versions of the singularity URIs for Seqera Containers. It's a bit of a pain, but here's how: https://nfcore.slack.com/archives/CJRH30T6V/p1729023924708159?thread_ts=1729023915.530259&cid=CJRH30T6V